server/src/test/java/org/spine3/server/entity/storagefield/StorageFieldTraverserShould.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2017, TeamDev Ltd. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package org.spine3.server.entity.storagefield;
+
+import com.google.protobuf.Message;
+import org.junit.Test;
+import org.spine3.protobuf.AnyPacker;
+import org.spine3.server.entity.StorageFields;
+import org.spine3.test.entity.Project;
+import org.spine3.testdata.Sample;
+
+import javax.annotation.Nullable;
+
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+/**
+ * @author Dmytro Dashenkov.
+ */
+public class StorageFieldTraverserShould {
+
+    @Test
+    public void traverse_over_storage_fields() {
+        final String intKey = "Int_value";
+        final String longKey = "Long_value";
+        final String stringKey = "String_value";
+        final String boolKey = "Bool_value";
+        final String floatKey = "Float_value";
+        final String doubleKey = "Double_value";
+        final String message1Key = "Message_value";
```

Comment:
Please don't ever use `message1` and `message2` as variable names or variable name parts.